app/assets/stylesheets/_sponsor_application.scss

```diff
@@ -0,0 +1,80 @@
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+}
+
+.steps-container {
+  overflow: hidden;
+  margin: 0px;
+  padding: 0px;
+  white-space: nowrap;
+  border-left: 2px solid;
+  border-right: 2px solid;
+  width: 100%;
+  counter-reset: steps;
+}
+.steps {
+  position: relative;
```

Comment:
Properties should be ordered counter-increment, cursor, display, height, left, line-height, margin-left, margin-right, min-height, position, transition, width